modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -65,10 +65,7 @@ def get_appeal
         begin
           appeal = appeal_adapter.parse(appeal) if Flipper.enabled?(:mobile_appeal_model, @current_user)
         rescue => e
-          PersonalInformationLog.create!(
-            data: { appeal:, error: e.message },
-            error_class: 'MobileAppealModelValidationError'
-          )
+          Rails.logger.info('MOBILE APPEAL VALIDATION ERROR', error: e.message)
```

Comment:
Querying PersonalInformationLog was potentially crashing pods and not finishing query, Changing to rails logger and just not putting any PII in the log